src/libcore/mem.rs

```diff
@@ -434,6 +434,11 @@ pub fn replace<T>(dest: &mut T, mut src: T) -> T {
 /// While this does call the argument's implementation of `Drop`, it will not
 /// release any borrows, as borrows are based on lexical scope.
 ///
+/// This effectively does nothing for
+/// [types which implement `Copy`](http://doc.rust-lang.org/nightly/book/ownership.html#copy-types),
+/// e.g. integers. Such values are copied and _then_ moved into the function,
+/// so the new copy persists after this function call.
```

Comment:
This disagrees slightly with the examples below where they state the copy is moved and dropped. Here it states the *the new copy persists* which is kinda the opposite. They should at least agree.

I don't know which way it's implemented or if it matters but from a explanation standpoint, letting someone copy your item so they have a new copy is more typical. It'd be more unusual if you let them copy and kept the new copy leaving them the ratty old copy you have (imagine copying homework or recipes for example).